src/services/shortLink/test/index.test.ts

```diff
@@ -0,0 +1,552 @@
+
+        const response = await injectPost(
+          app,
+          mockShortLinkPayload({ item_id: item.id, alias: '1_2$3<45' }),
+        );
+        expect(response.statusCode).toEqual(StatusCodes.BAD_REQUEST);
+      });
+
+      it('Bad request if post short links without body', async () => {
+        const response = await injectPost(app);
+        expect(response.statusCode).toEqual(StatusCodes.BAD_REQUEST);
+      });
+
+      it('Bad request if post short links with empty body', async () => {
+        const response = await injectPost(app, {});
+        expect(response.statusCode).toEqual(StatusCodes.BAD_REQUEST);
+      });
+    });
+
+    describe('Not Found', () => {
+      it('Conflict if post short links with already exist alias', async () => {
+        const { item } = await mockPostData({ author: actor });
+        const { item: item2 } = await mockPostData({ author: actor });
+
+        await injectPost(app, mockShortLinkPayload({ item_id: item.id }));
+
+        const response = await injectPost(app, mockShortLinkPayload({ item_id: item2.id }));
+        expectException(response, new ShortLinkDuplication(MOCK_ALIAS));
+      });
+    });
+
+    describe('Not Found', () => {
+      it('Not found if post short links with invalid item id', async () => {
+        const response = await injectPost(app, mockShortLinkPayload());
+        expect(response.statusCode).toEqual(StatusCodes.NOT_FOUND);
+
+        const check = await injectGet(app, MOCK_ALIAS);
+        expect(check.statusCode).toEqual(StatusCodes.NOT_FOUND);
+      });
+    });
+
+    describe('Authorized Posts', () => {
+      afterEach(async () => {
+        const check = await injectGet(app, MOCK_ALIAS);
+        expect(check.statusCode).toEqual(StatusCodes.OK);
+        expect(check.json().alias).toEqual(MOCK_ALIAS);
+      });
+
+      it('Success if post short links with author member', async () => {
+        const { item } = await mockPostData({ author: actor });
+
+        const response = await injectPost(app, mockShortLinkPayload({ item_id: item.id }));
+        expect(response.statusCode).toEqual(StatusCodes.OK);
+        expect(response.json().alias).toEqual(MOCK_ALIAS);
+      });
+
+      it('Success if post short links with admin permission', async () => {
+        const { item } = await mockPostData({
+          author: bob,
+          member: actor,
+          permission: PermissionLevel.Admin,
+        });
+
+        const response = await injectPost(app, mockShortLinkPayload({ item_id: item.id }));
+        expect(response.statusCode).toEqual(StatusCodes.OK);
+        expect(response.json().alias).toEqual(MOCK_ALIAS);
+      });
+    });
+  });
+
+  describe('GET /short-links/:itemId', () => {
+    beforeEach(async () => {
+      ({ app, actor } = await build());
+      bob = await saveMember(BOB);
+    });
+
+    it('Not found if get short links with invalid item id', async () => {
+      const response = await injectGet(app, MOCK_ALIAS);
```

Comment:
Maybe here use a string like `bubu` instead, to amke it clear that it is not a uuid.

Reply:
The problem is if I'm not using a good UUID format, I will have a bad request. What I want to test here is to be sure to receive a 404 error if an itemId not exist, and not the format of the given UUID.

Reply:
The problem is if I'm not using a good UUID format, I will have a bad request. What I want to test here is to be sure to receive a 404 error if an itemId not exist, and not the format of the given UUID.